plugins/GSdx/GS.h

```diff
@@ -1425,3 +1425,25 @@ enum class Filtering : uint8
 	Trilinear_Bilinear_Forced,
 	Trilinear_Always
 };
+
+//Enum order matters! Don't change it!
+enum class CRCHackLevel : int8
+{
+	Automatic = -1,
+	None = 0,
+	Minimum = 1,
+	Partial = 2,
+	Full = 3,
+	Aggressive = 4
+};
+
+static inline CRCHackLevel AutoSelectCRCHackLevel(GSRendererType renderer)
+{
+	switch (renderer)
+	{
+	case GSRendererType::OGL_HW:
+		return CRCHackLevel::Partial;
+	default:
+		return CRCHackLevel::Full;
+	}
```

Comment:
``switch-case`` doesn't seem ideal here, a ternary operator should suffice. `` return (renderer == GSRendererType::OGL_HW) ? CRCHackLevel::Partial : return CRCHackLevel::Full``. **(or)** maybe a simple ``if()`` check might be enough.